examples/04_pong/main.rs

```diff
@@ -82,8 +77,20 @@ fn run() -> Result<()> {
         .with_bundle(TransformBundle::new().with_dep(&["ball_system", "paddle_system"]))?
         .with_bundle(AudioBundle::new(|music: &mut Music| music.music.next()))?
         .with_bundle(RenderBundle::new())?
-        .with_local(RenderSystem::build(pipe, Some(display_config))?);
-    Ok(game.build()?.run())
+        .with_bundle(UiBundle::new(&[]))?;
+
+    let pipe = {
```

Comment:
This makes me sad. I don't know how to fix it though :/

Reply:
Basically by fixing the `ApplicationBuilder` I guess.